package.json

```diff
@@ -12,7 +12,9 @@
     "url": "https://github.com/krakenjs/engine-munger.git"
   },
   "dependencies": {
+    "adaro": "~0.1.5",
     "bl": "^0.9.0",
+    "dustjs-linkedin": "~2.6.2",
```

Comment:
Correct-- but that's what the peerdependency was expressing.

dust also relies on sharing an instance for helpers.